project/Dependencies.scala

```diff
@@ -18,10 +18,10 @@ import sbt._
 
 object Version {
   val scala       = "2.11.12"
```

Comment:
GeoSpark was having some issues compiling with Scala 2.12 when I tried, although now that I look at their releases I think it might also need me to upgrade Spark to 3.0. I'll try upgraded all these things together and see if they work.

I forgot about the requester-pays options - I'll look into adding that as well